nbextensions/bulkImportCell/tabs/configure.js

```diff
@@ -21,7 +22,9 @@ define([
             cellBus = options.bus, // the bus to communicate with the main widget
             FILE_PATH_TYPE = 'filePaths',
             PARAM_TYPE = 'params';
-        let container = null;
+        let container = null,
+            filePathWidget,
+            paramsWidget;
```

Comment:
Need to keep track of the widgets and actually propagate the `stop` command to them later.